bridge/slack/slack.go

```diff
@@ -317,7 +317,7 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 	return b.postMessage(&msg, channelInfo)
 }
 
-func (b *Bslack) updateTopicOrPurpose(msg *config.Message, channelInfo *slack.Channel) (bool, error) {
```

Comment:
Removing the returned `bool` as suggested by the `unparam` linter. It's value is always `true`.